toonz/sources/toonzlib/preferences.cpp

```diff
@@ -255,6 +255,8 @@ Preferences::Preferences()
     , m_fitToFlipbookEnabled(false)
     , m_previewAlwaysOpenNewFlipEnabled(false)
     , m_autosaveEnabled(false)
+    , m_autosaveSceneEnabled(true)
+    , m_autosaveOtherFilesEnabled(false)
```

Comment:
It is quite understandable that most of users would prefer to save everything rather than to save only scene.
Sorry to trouble you but it is OK to set this initial value to `true`. I will add a note of caution to the next version's release notes regarding this change.